2015/9-easy-exceptions-and-logging.pod

```diff
@@ -0,0 +1,293 @@
+=encoding utf8
+
+=head1 Exception handling, error logging and user messages made easy
+
+=head2 Introduction
+
+Have you ever experienced any of the following when writing your Dancer
+application?
+
+=over 1
+
+=item * Found that handling user errors involved writing a lot of code
+
+=item * Wanted to log messages from a module, without that module depending on Dancer2
+
+=item * Wanted flexibility on sending messages to the user, to the system, or both
+
+=item * Found yourself writing the same code in every application in order to send messages to a template
+
+=item * Wanted a clean way to handle 500 errors, whilst getting notified as the system administrator
+
+=back
+
+L<Dancer2::Plugin::LogReport> combined with L<Dancer2::Logger::LogReport>
+are designed to do all of the above, and more.
+Although the 2 modules can be used separately, it's recommended to use
+them both together.
+
+
+=head2 Getting Started
+
+Let's get started by adding the 2 modules to a Dancer2 app:
+
+ # In your route handler
+ package MyApp;
+ use Dancer2;
+ use Log::Report ();
+ use Dancer2::Plugin::LogReport;
+
+ # In your config
+ logger: LogReport
+
+So, what does this give us as standard? Let's have a look:
+
+=over 4
+
+=item * Default console logging
+
+Without any configuration, the default logging is unchanged. That is, messages
+will continue to be sent to the console via STDERR (the formatting has changed,
+but we'll worry about that later). So, not really any difference there,
```

Comment:
Thanks, that's better. I've worded slightly differently, as the formatting is the thing we "worry" about later, and isn't actually covered in this intro (maybe it should be)